### PR TITLE
[IMP] 14.0 rename SaleOrderButton to align with Odoo

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

File: pos_sale_order/static/src/xml/SaleOrderWidget.xml [hunk 1]
```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <t t-name="SaleOrderWidget" owl="1">
-        <div class="ticket-button">
-            <a
-                class="fa fa-home"
-                style="color:white"
-                href="/web#view_type=list&amp;model=sale.order&amp;action=pos_sale_order.action_sale_order_pos"
```
Comment (hparfr) [inline pos_sale_order/static/src/xml/SaleOrderWidget.xml]: This is not "just a rename".
Where is this link implemented ?

Comment (hparfr) [reply pos_sale_order/static/src/xml/SaleOrderWidget.xml]: By the way, this should not be here. This delivery button should be put elsewhere. Like pos_sale_order_delivery

Comment (sebastienbeau) [reply pos_sale_order/static/src/xml/SaleOrderWidget.xml]: @PierrickBrun Indeed it's broken, can you check ?

Comment (PierrickBrun) [reply pos_sale_order/static/src/xml/SaleOrderWidget.xml]: I removed this part. It does not work and in any case it should be done on another PR (the one the will put this in pos_sale_order_delivery for instance)

**Pull request merged**